core/scrollbar.js

```diff
@@ -181,6 +179,7 @@ Blockly.ScrollbarPair.prototype.getRatio_ = function(knobValue, barLength) {
  * Class for a pure SVG scrollbar.
  * This technique offers a scrollbar that is guaranteed to work, but may not
  * look or behave like the system's scrollbars.
+ * NB: "Knob" and "handle" are used interchangeably in this file.
```

Comment:
TBH, mostly fear that someone, somewhere is using private properties of the scrollbar.